pipewrench/merge.py

```diff
@@ -199,7 +179,13 @@ def render(template, **kwargs):
     :return: The reified template
     """
     template = Template(template)
-    template_functions = [map_datatypes, dumps, map_clobs, order_columns]
+    template_functions = [dumps]
+    plugin_path = os.path.join(os.path.dirname(__file__), 'plugins')
```

Comment:
plugins is a dir at the same level as merge.py...

Reply:
@afoerster  I have put the plugins path to be relative to 'merge.py'  ----os.path.dirname(__file__)+/plugins, where os.path.dirname(__file__) is the directory containing 'merge/py', is this the right assumption?

Reply:
It should be relative to the `--pipeline-templates` argement (`pipeline_templates_dir`), this way a user can add a new plugin to their templates and it will automatically be picked up. merge.py is hidden way out in some directory somewhere, like venv/python2.7/site-packages/pipewrench/....

Reply:
Makes sense. Will update the dir structure.